src/components/standup/standupContainer.module.scss

```diff
@@ -0,0 +1,152 @@
+$theme-focusHighlight: #1c64f2;
+$theme-primaryBackground: #1c325e;
+$theme-whiteColor: #ffffff;
+$theme-bannerFontColor: #f1416c;
+$theme-secondaryBackground: #1e429f;
+$theme-CeriseRed: #d61f69;
+$theme-headingColor: #6b7280;
+$theme-inputFieldBorder: #d1d5db;
+$theme-nonActiveButtonBackground: #e5e7eb;
+$radius: 0.5rem;
+
+.standupContainer {
+    display: flex;
+    flex-direction: column;
+    margin-top: 1.88rem;
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+}
+
+.standupBanner {
+    background-color: $theme-primaryBackground;
+    width: 84%;
+    height: 12%;
```

Comment:
why not go with 80 and 10?

Reply:
because as per Figma width  and height are 1339 and 192px and 1339/16 = 83.63 and 192/16=12 px and that's why 84% and 12%

Reply:
why should 80 and 10 why not 84 and 12